src/abstractions/FunctionsAuthenticationBuilderExtensions.cs

```diff
@@ -11,6 +11,21 @@ namespace Microsoft.Extensions.DependencyInjection
 
     public static class FunctionsAuthenticationBuilderExtensions
     {
+        /// <summary>
+        /// Adds the JWT "FunctionsBearer" scheme to the authentication configuration.
+        /// </summary>
+        /// <param name="builder">The current authentication builder.</param>
+        /// <param name="configureOptions">An action configuring the JWT options for authentication. 
+        /// <param name="authenticationScheme">The authentication scheme to use.</param>"
+        /// <returns>A instance of the <see cref="FunctionsAuthenticationBuilder"/></returns>
+        public static FunctionsAuthenticationBuilder AddJwtFunctionsBearer(
+            this FunctionsAuthenticationBuilder builder, string authenticationScheme, Action<JwtBearerOptions> configureOptions)
+        {
+            builder.AddJwtBearer(authenticationScheme, configureOptions);
```

Comment:
Can we add a check to ensure the specified authentication scheme is not `Bearer`?